tests/test_django/app/migrations/0001_initial.py

```
@@ -1,4 +1,4 @@
-# Generated by Django 5.1.4 on 2025-04-04 05:32
+# Generated by Django 5.1.4 on 2025-04-30 06:27
```

Ruff (0.8.2)

Missing docstring in public module

(D100)